core/chains/evm/gas/block_history_estimator_test.go

```diff
@@ -1459,17 +1459,21 @@ func TestBlockHistoryEstimator_IsUsable(t *testing.T) {
 		assert.Equal(t, true, bhe.IsUsable(tx, block, cfg.ChainType(), geCfg.PriceMin(), logger.Test(t)))
 	})
 
-	t.Run("returns false if transaction is of type 0x7c or 0x7b only on Celo", func(t *testing.T) {
+	t.Run("returns false if transaction is of type 0x7c, 0x7b, or 0x7e only on Celo", func(t *testing.T) {
```

Comment:
Can we not change the chain type to OptimismBedrock, so this automatically is included?

Reply:
Good point. Tested the config using optimismBedrock chaintype today and it avoids this issue. Once Celo mainnet migrates we can deprecate the celo chaintype.